locations/spiders/subway_kr.py

```diff
@@ -0,0 +1,43 @@
+import chompjs
+import scrapy
+from scrapy import Request
+from scrapy.http import Response
+from scrapy.linkextractors import LinkExtractor
+
+from locations.dict_parser import DictParser
+from locations.spiders.subway import SubwaySpider
+from locations.spiders.vapestore_gb import clean_address
+
+
+class Subwaykrpider(scrapy.Spider):
+    name = "subway_kr"
+    item_attributes = SubwaySpider.item_attributes
+    link_extractor = LinkExtractor(allow="/storeDetail?")
+
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+
+    def start_requests(self):
+        yield Request("https://www.subway.co.kr/storeSearch?page=1", meta={"page": 1})
+
+    def parse(self, response: Response, **kwargs):
+        links = self.link_extractor.extract_links(response)
+        if len(links) == 1:
+            return
```

Comment:
Is this an artifact of the last page currently only having 1 time? Will this still work when they add 1 more POI?

https://www.subway.co.kr/storeSearch?page=59

Reply:
Good catch. I will fix.

Reply:
Done